doc/user_manual/PostProcessors/CrossValidation.tex

```diff
@@ -7,7 +7,7 @@ \subsubsection{CrossValidation}
 rounds such that each data point has a chance of being validated against the various sets. The basic form of
 cross-validation is k-fold cross-validation. Other forms of cross-validation are special cases of k-fold or involve
 repeated rounds of k-fold cross-validation. \nb It is important to notice that this post-processor currently can
-only accept \textbf{PointSet} data object.
+only accept \textbf{PointSet} data object and untrained ROM. If the ROM is already trained, an error will be raised.
```

Comment:
Issue #721 metnioned the ROM is wiped from this PP, is this something that can be avoided?